setup.py

```diff
@@ -34,6 +35,9 @@ def read(*filenames, **kwargs):
             'flake8_quotes = flake8_quotes:QuoteChecker',
         ],
     },
+    license='MIT',
+    zip_safe=False,
```

Comment:
I wasn't sure if the tests were zip safe. If they read files from disk, they would potentially break in a zip file.  A quick test install as a zipped egg would confirm it.